daprdocs/content/en/reference/components-reference/supported-bindings/kubemq.md

```diff
@@ -0,0 +1,118 @@
+---
+type: docs
+title: "KubeMQ binding spec"
+linkTitle: "KubeMQ"
+description: "Detailed documentation on the KubeMQ binding component"
+aliases:
+  - "/operations/components/setup-bindings/supported-bindings/kubemq/"
+---
+
+## Component format
+
+To setup KubeMQ binding create a component of type `bindings.kubemq`. See [this guide]({{< ref "howto-bindings.md#1-create-a-binding" >}}) on how to create and apply a binding configuration.
+
+
+```yaml
+apiVersion: dapr.io/v1alpha1
+kind: Component
+metadata:
+  name: binding-topic
+spec:
+  type: bindings.kubemq
+  version: v1
+  metadata:
+    - name: address
+      value: localhost:50000
+    - name: channel
+      value: queue1
+```
+
+## Spec metadata fields
+
+| Field              | Required | Details                                                      | Example                                |
+|--------------------|:--------:|--------------------------------------------------------------|----------------------------------------|
+| address            |    Y     | Address of the KubeMQ server                                 | `"localhost:50000"`                    |
+| channel            |    Y     | The Queue channel name                                       | `queue1`)                              |
+| authToken          |    N     | Auth JWT token for connection                                | `ew...`                                |
+| autoAcknowledged   |    N     | Sets if received queue message is automatically acknowledged | `true` or `false` (default is `false`) |
+| pollMaxItems       |    N     | Sets the number of messages to poll on every connection      | `1`                                    |
+| pollMaxItems       |    N     | Sets the number of messages to poll on every connection      | `1`                                    |
+| pollTimeoutSeconds |    N     | Sets the time in seconds for each poll interval              | `3600`                                 |
+
+## Binding support
+
+This component supports both **input and output** binding interfaces.
+
```

Comment:
Need to list the available operations for example `create`, `get` etc. See example here. https://docs.dapr.io/reference/components-reference/supported-bindings/blobstorage/#binding-support
Strongly suggest creating examples of each so that people can cut and paste.

Reply:
You are write,
removed the duplicated line